components/announcement_bar/index.js

```diff
@@ -34,6 +36,7 @@ function mapStateToProps(state) {
         canViewSystemErrors,
         latestError,
         totalUsers,
+        warnMetricsStatus,
```

Comment:
I don't see where this prop is used in the `AnnouncementBar` component. I think the changes in this file aren't needed.

Reply:
It is passed to the `ConfigurationAnnouncementBar` through the `AnnouncementBarController` and used in it to determine the show/not show off the `AnnouncementBar`. Note that this connects to `AnnouncementBarController` not `AnnouncementBar` component.

Reply:
:+1: